interface/forms/track_anything/history.php

```diff
@@ -69,8 +69,7 @@
 
 <?php require $GLOBALS['srcdir'] . '/js/xl/dygraphs.js.php'; ?>
 
-<!-- TODO: Why is there no Bootstrap here??? !-->
-<?php Header::setupHeader(['no_bootstrap', 'no_fontawesome', 'no_textformat', 'no_dialog', 'dygraphs']); ?>
+<?php Header::setupHeader(['no_fontawesome', 'no_textformat', 'no_dialog', 'dygraphs']); ?>
```

Comment:
I'll look into the the bootstrap and dygraphs issue (ie. i'll test to ensure graphing works fine with the bootstrap inluded).

Reply:
just brought in commit that cleaned up the bootstrap/dygraphs stuff